### PR TITLE
oberon-risc-emu: init at 2020-08-18

### PR DESCRIPTION
**Pull request by siraben**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/emulators/oberon-risc-emu/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub, SDL2 }:
+
+stdenv.mkDerivation {
+  pname = "oberon-risc-emu";
+  version = "unstable-2020-08-18";
```
Comment (IvarWithoutBones) [inline pkgs/misc/emulators/oberon-risc-emu/default.nix:5]: `unstable` should be apart of `pname`, not version. See https://github.com/NixOS/nixpkgs/issues/68518
```suggestion
  pname = "oberon-risc-emu-unstable";
  version = "2020-08-18";
```

File: pkgs/misc/emulators/oberon-risc-emu/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    repo = "oberon-risc-emu";
+    rev = "26c8ac5737c71811803c87ad51f1f0d6e62e71fe";
+    sha256 = "1iriix3cfcpbkjb5xjb4ysh592xppgprwzp3b6qhwcx44g7kdvxq";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv risc $out/bin
+  '';
+}
```
Comment (IvarWithoutBones) [inline pkgs/misc/emulators/oberon-risc-emu/default.nix:20]: Seems like the `meta` section is missing, which is required.

**Review by IvarWithoutBones — CHANGES_REQUESTED** [review]
A few suggestions, but functionality wise this PR LGTM :+1: 

Binary builds, and successfully displays the help section.

Result of `nixpkgs-review pr 100042` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>oberon-risc-emu</li>
  </ul>
</details>

**Comment by siraben** [comment]
@GrahamcOfBorg eval

File: pkgs/misc/emulators/oberon-risc-emu/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  buildInputs = [ SDL2 ];
+
+  src = fetchFromGitHub {
+    owner = "pdewacht";
+    repo = "oberon-risc-emu";
+    rev = "26c8ac5737c71811803c87ad51f1f0d6e62e71fe";
+    sha256 = "1iriix3cfcpbkjb5xjb4ysh592xppgprwzp3b6qhwcx44g7kdvxq";
+  };
+
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/oberon-risc-emu/default.nix:15]:
```suggestion
  src = fetchFromGitHub {
    owner = "pdewacht";
    repo = "oberon-risc-emu";
    rev = "26c8ac5737c71811803c87ad51f1f0d6e62e71fe";
    sha256 = "1iriix3cfcpbkjb5xjb4ysh592xppgprwzp3b6qhwcx44g7kdvxq";
  };

  buildInputs = [ SDL2 ];

```

File: pkgs/misc/emulators/oberon-risc-emu/default.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, SDL2 }:
+
+stdenv.mkDerivation {
+  pname = "oberon-risc-emu-unstable";
+  version = "2020-08-18";
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/oberon-risc-emu/default.nix:5]:
```suggestion
  pname = "oberon-risc-emu";
  version = "unstable-2020-08-18";
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 100042` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>oberon-risc-emu</li>
</ul>
</details>

**Pull request merged**